models/labels/cex/labels_cex_ethereum.sql

```diff
@@ -1,496 +1,12 @@
-{{config(alias='cex_ethereum')}}
+{{config(alias='cex_ethereum',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "sector",
+                                    "labels",
+                                    \'["hildobby"]\') }}')}}
 
 SELECT blockchain, address, name, category, contributor, source, created_at, updated_at
 FROM (VALUES
     -- Binance, Source: https://etherscan.io/accounts/label/binance
     (array('ethereum'),'0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be', 'Binance 1', 'cex', 'hildobby', 'static', timestamp('2022-08-28'), now())
     , (array('ethereum'), '0xd551234ae421e3bcba99a0da6d736074f22192ff', 'Binance 2', 'cex', 'hildobby', 'static', timestamp('2022-08-28'), now())
-    , (array('ethereum'), '0x564286362092d8e7936f0549571a803b203aaced', 'Binance 3', 'cex', 'hildobby', 'static', timestamp('2022-08-28'), now())
```

Comment:
Why drop this?

Reply:
@aalan3 it shouldn't have been, thanks for spotting this !